pages/DSNP/Types/Update.md

```diff
@@ -0,0 +1,45 @@
+# Update Announcement
+
+An Update Announcement is a way to note intent to update previously
+announced content.
+
+## Fields
+
+| Field | Description | Data Type | Serialization | Parquet Type | Bloom Filter |
+| ----- | ----------- | --------- | ------------- | ------------ | ------------ |
```

Comment:
based on https://github.com/LibertyDSNP/spec/issues/145 we probably don't need the signature field